clarity/src/vm/ast/parser/v1.rs

```diff
@@ -1300,4 +1316,13 @@ mod test {
             }
         });
     }
+
+    #[test]
+    fn test_long_contract_name() {
+        let long_contract_name = "(define-private (transfer (id uint) (receiver principal)) (contract-call? 'SP3D6PV2ACBPEKYJTCMH7HEN02KP87QSP8KTEH335.megapont-robot-expansion-nftSPNWZ5V2TPWGQGVDR6T7B6RQ4XMGZ4PXTEE0VQ0S.guests-hosted-stacks-parrots transfer id tx-sender receiver))";
+        assert!(match ast::parser::v1::parse(long_contract_name).unwrap_err().err {
+            ParseErrors::SeparatorExpected(_) => true,
+            _ => false,
+        });
+    }
```

Comment:
Can you also add the test for the case seen here, if it's not in the codebase already? https://github.com/stacks-network/stacks-blockchain/issues/3133

Reply:
Yes, that is here already - https://github.com/stacks-network/stacks-blockchain/blob/2894afd5767a4d32a5e4be5132abcaa04a4b47b4/src/chainstate/stacks/db/transactions.rs#L2409-L2412

Reply:
Thanks!